zebra-chain/src/sapling/shielded_data.rs

```diff
@@ -199,7 +199,7 @@ where
         // TODO: use TransferData::shared_anchor to improve performance for V5 transactions
         self.spends_per_anchor()
             .map(|spend| spend.per_spend_anchor)
-            .sorted()
+            .unique_by(|raw| raw.0.to_bytes())
             .dedup()
```

Comment:
`unique_by()` does the same thing as `sorted().dedup()`, so we don't need the `dedup()` any more

```suggestion
            .unique_by(|raw| raw.0.to_bytes())
```

Reply:
Thanks, that makes sense, removed at https://github.com/ZcashFoundation/zebra/pull/3544/commits/55504246e6cd26a6610c3caa4823527635684655